src/main/java/org/primefaces/cache/HazelcastCacheProvider.java

```diff
@@ -26,50 +26,50 @@
 import com.hazelcast.config.Config;
 import com.hazelcast.core.Hazelcast;
 import com.hazelcast.core.HazelcastInstance;
-import com.hazelcast.core.IMap;
+import com.hazelcast.map.IMap;
 
 public class HazelcastCacheProvider implements CacheProvider {
 
-    private final HazelcastInstance hazelcastInstance;
+	private final HazelcastInstance hazelcastInstance;
 
```

Comment:
We typically don't format files in PR's. Can you make this PR just the bare minimum changes?